support/ci/appveyor.ps1

```diff
@@ -64,28 +64,73 @@ if (Test-SourceChanged -or (test-path env:HAB_FORCE_TEST)) {
                 popd
             }
         }
+        elseif ($BuildAction -like 'package') {
+            Write-Host "Download and install latest release of hab.exe"
+            $bootstrapDir = "c:\habitat"
+            $url = "https://api.bintray.com/content/habitat/stable/windows/x86_64/hab-%24latest-x86_64-windows.zip?bt_package=hab-x86_64-windows"
+            mkdir $bootstrapDir -Force
+            # download a hab binary to build hab from source in a studio
+            Invoke-WebRequest -UseBasicParsing -Uri $url -OutFile hab.zip
+            Expand-Archive -Path hab.zip -DestinationPath $bootstrapDir
+            $habExe = (Get-Item "$bootstrapDir/*/hab.exe").FullName
+            
+            # This will override plan's CARGO_TARGET_DIR so we do not have to build each clean
+            $env:HAB_CARGO_TARGET_DIR = "c:\projects\habitat\target"
+
+            $env:HAB_ORIGIN="core"
+            if($env:ORIGIN_KEY) {
+                "SIG-SEC-1`ncore-20160810182414`n`n$($env:ORIGIN_KEY)" | & $habExe origin key import
```

Comment:
Any reason the whole key isn't in the environment?  Once that key needs to be rev'd there'll be two places to fix that.